src/java.base/share/classes/java/lang/reflect/InvocationHandler.java

```diff
@@ -262,33 +261,6 @@ public static Object invokeDefault(Object proxy, Method method, Object... args)
             throws Throwable {
         Objects.requireNonNull(proxy);
         Objects.requireNonNull(method);
-
```

Comment:
This removed piece of code is moved to `Proxy.invokeDefault` (which is now called at the end of this removed code). The code moved is mostly the same, except it checks if the caller object is null. If it is  null, then it won't execute anything.